--- origsrc/gengetopt-2.23/tests/test_conf_parser_ov2.c	2019-06-04 10:28:20.000000000 +0200
+++ src/gengetopt-2.23/tests/test_conf_parser_ov2.c	2020-11-29 19:50:13.222312200 +0100
@@ -31,7 +31,7 @@ main (int argc, char **argv)
 
   /* call the config file parser */
   if (test_conf_parser_cmd_parser_config_file
-      ("../../tests/test_conf2.conf", &args_info, params) != 0)
+      ("/usr/local/src/cygwin-gengetopt/gengetopt-2.23-1.x86_64/src/gengetopt-2.23/tests/test_conf2.conf", &args_info, params) != 0)
     exit(1);
 
   /* 
--- origsrc/gengetopt-2.23/tests/test_conf_parser_ov3.c	2019-06-04 10:28:20.000000000 +0200
+++ src/gengetopt-2.23/tests/test_conf_parser_ov3.c	2020-11-29 19:50:13.543950100 +0100
@@ -22,12 +22,12 @@ main (int argc, char **argv)
 
   /* do not override command line options, initialize args_info */
   if (test_conf_parser_cmd_parser_configfile
-      ("../../tests/test_conf2.conf", &args_info, 0, 1, 0) != 0)
+      ("/usr/local/src/cygwin-gengetopt/gengetopt-2.23-1.x86_64/src/gengetopt-2.23/tests/test_conf2.conf", &args_info, 0, 1, 0) != 0)
     exit(1);
 
   /* do not override, do not initialize args_info */
   if (test_conf_parser_cmd_parser_configfile
-      ("../../tests/test_conf3.conf", &args_info, 0, 0, 0) != 0)
+      ("/usr/local/src/cygwin-gengetopt/gengetopt-2.23-1.x86_64/src/gengetopt-2.23/tests/test_conf3.conf", &args_info, 0, 0, 0) != 0)
     exit(1);
 
   if (test_conf_parser_cmd_parser2 (argc, argv, &args_info, 1, 0, 1) != 0)
--- origsrc/gengetopt-2.23/tests/test_conf_parser_ov4.c	2019-06-04 10:28:20.000000000 +0200
+++ src/gengetopt-2.23/tests/test_conf_parser_ov4.c	2020-11-29 19:50:13.856512300 +0100
@@ -34,7 +34,7 @@ main (int argc, char **argv)
 
   /* call the config file parser */
   if (test_conf_parser_cmd_parser_config_file
-      ("../../tests/test_conf2.conf", &args_info, params) != 0) {
+      ("/usr/local/src/cygwin-gengetopt/gengetopt-2.23-1.x86_64/src/gengetopt-2.23/tests/test_conf2.conf", &args_info, params) != 0) {
     result = 1;
     goto stop;
   }
